apps/onlyoffice.sh

```diff
@@ -21,31 +18,58 @@ debug_mode
 root_check
 
 # Nextcloud 13 is required.
-lowest_compatible_nc 13
+lowest_compatible_nc 18
 
 # Test RAM size (2GB min) + CPUs (min 2)
 ram_check 2 OnlyOffice
 cpu_check 2 OnlyOffice
 
-# Notification
-msg_box "Before you start, please make sure that port 80+443 is directly forwarded to this machine!"
-
 # Get the latest packages
 apt update -q4 & spinner_loading
 
-# Check if Nextcloud is installed
-print_text_in_color "$ICyan" "Checking if Nextcloud is installed..."
-if ! curl -s https://"${NCDOMAIN//\\/}"/status.php | grep -q 'installed":true'
+# Check if onlyoffice is already installed
+print_text_in_color "$ICyan" "Checking if Onlyoffice is already installed..."
+if version_gt "$CURRENTVERSION" "18.0.1" && ! does_this_docker_exist 'onlyoffice/documentserver'
+then
+    install_if_not jq
+    if occ_command_no_check app:list --output=json | jq -e '.enabled | .documentserver_community' > /dev/null
+    then
+        choice=$(whiptail --radiolist "It seems like 'Onlyoffice' is already installed.\nChoose what you want to do.\nSelect by pressing the spacebar and ENTER" "$WT_HEIGHT" "$WT_WIDTH" 4 \
```

Comment:
I think we should decide first, if a whiptail is the way to go here, since we could do it like I have suggested it here: https://github.com/nextcloud/vm/pull/1072#issuecomment-586652048